### DISCUSSION TITLE
This IDE build has expired. Please download a new build from JetBrains official site

### DISCUSSION THREAD

**Discussion by PHPirates** [JetBrains/Qodana#117] [Q&A] [answered]
I'm using Qodana in GitHub Actions, I had it set up a while ago but now the log shows

```
✓  Pulled the latest version of linter
/opt/hostedtoolcache/qodana/2022.2.1/x64/qodana scan --skip-pull --cache-dir /home/runner/work/_temp/qodana/caches --results-dir /home/runner/work/_temp/qodana/results 
This IDE build has expired. Please download a new build from JetBrains official site
2022/08/14 10:38:24 IDEA process exit code: 7
2022/08/14 10:38:24 Sync IDEA cache from: /data/project/.idea to: /data/cache/.idea
✗  Qodana exited with code 7
```

I don't know if this is an issue with qodana-action or how it uses Qodana, but I don't know why it's caching (I suppose) an expired IDE build, and I don't know of any way to empty the cache.

Build log: https://github.com/Hannah-Sten/TeXiFy-IDEA/runs/7825843814?check_suite_focus=true
I'm using `JetBrains/qodana-action@v2022.2.1` without any options set.

**Comment by brichbash** [comment, marked as answer]
Hello,
You have `qodana.yaml` in your repository with set linter `jetbrains/qodana-jvm:2021.3-eap`. Please change it to `jetbrains/qodana-jvm:2022.2-eap`. That should help.

**Reply by PHPirates** [reply]
Oh, I had completely forgotten that I had configured it like that. Thank you so much for your help!
